tests/unit/mixins/active-workflows-test.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+import ActiveWorkflowsMixin from 'bespin-ui/mixins/active-workflows';
+import { module, test } from 'qunit';
+
+module('Unit | Mixin | active workflows');
+
+test('model method returns all active workflows sorted by name', function(assert) {
+  let ActiveWorkflowsObject = Ember.Object.extend(ActiveWorkflowsMixin);
+  let subject = ActiveWorkflowsObject.create({
+    store: {
+      findAll(recordModel) {
+        return {
+          then(filterFunc) {
```

Comment:
I suggest that `findAll(recordModel)` could return a promise with `Ember.RSVP.resolve()` instead of the `then`/`filterFunc` construction.